src/Haskell/Ide/Engine/Plugin/Pragmas.hs

```diff
@@ -53,15 +58,22 @@ addPragmaCmd = CmdSync $ \(AddPragmaParams uri pragmaName) -> do
 
 -- ---------------------------------------------------------------------
 
+-- | Offer to add a missing Language Pragma to the top of a file.
+-- Pragmas are defined by a curated list of known pragmas, see 'possiblePragmas'.
+-- May also offer to add unknown pragmas, if the pragma is a subexpression of
+-- an existing plugin.
```

Comment:
I do not understand the last two lines, can you try use a different wording.

Reply:
It was wrong anyways, thanks for making me look at it again.